plugin/builtin/replicacounttransformer/ReplicaCountTransformer_test.go

```diff
@@ -24,26 +24,130 @@ apiVersion: builtin
 kind: ReplicaCountTransformer
 metadata:
   name: notImportantHere
+
 replica:
-  name: deploy1
+  name: myapp
   count: 23
+fieldSpecs:
+- path: spec/replicas
+  create: true
+  kind: Deployment
+
+- path: spec/replicas
+  create: true
+  kind: ReplicationController
+
+- path: spec/replicas
+  create: true
+  kind: ReplicaSet
+
+- path: spec/replicas
+  create: true
+  kind: StatefulSet
 `, `
-group: apps
-apiVersion: v1
+apiVersion: builtin
+kind: Service
+metadata:
+  name: myapp
+spec:
+  ports:
+  - port: 1111
+  targetport: 1111
+---
+apiVersion: builtin
 kind: Deployment
 metadata:
-  name: deploy1
+  name: otherapp
+spec:
+  replicas: 5
+---
+apiVersion: builtin
+kind: Deployment
+metadata:
+  name: myapp
+spec:
+  replicas: 5
+---
+apiVersion: builtin
```

Comment:
Instead of `builtin`, use `apps/v1`
for your four resources
 - `Deployment`
- `ReplicationController`
- `ReplicaSet`
- `StatefulSet`


From
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/

The `builtin` thing applies to plugins only.